src/core/database.py

```diff
@@ -0,0 +1,78 @@
+# coding: utf8
+
+__contact__ = "info@hytech-imaging.fr"
+__copyright__ = "Copyright (c) 2021 Hytech Imaging"
+
+import os.path
+
+from PyQt5.QtWidgets import QMessageBox
+from qgis._core import (
+    QgsWkbTypes,
+    QgsFields,
+    QgsVectorFileWriter,
+    QgsCoordinateReferenceSystem,
+    QgsCoordinateTransformContext,
+    QgsFeatureSink,
+    QgsVectorLayer,
+    QgsProject,
+)
+
+CONST_DB_NAME = "sammo-boat.gpkg"
```

Comment:
We can remove the `CONST_` prefix. Upper case are constants by convention.

Reply:
Done commit #8 (33fc7e1847da170fdcc5362c7978609aace3250c)